### ISSUE TITLE
investigate generating static manifests and level-zero iiif images from Bodleian

### ISSUE THREAD

**Issue by kmcelwee** [Princeton-CDH/geniza#671]
Slack conversation: https://cdhprinceton.slack.com/archives/C02J9G5CFEE/p1646083355071689

* Example page to scrape: https://hebrew.bodleian.ox.ac.uk/catalog/volume_5
* To generate iiif asset: https://github.com/IIIF-Commons/biiif 
* Image url pattern is based on shelfmark https://genizah.bodleian.ox.ac.uk/fragments/full/MS_HEB_a_2_1a.jpg
* Potentially publish a kind of github pages static site with the collection
* The hard part may be getting the biiif manifests to work with our code

- [x] Make small-scale attempt to make sure that investment is worth it.

**Comment by rlskoeser** [comment]
Note that biiif generates presentation api 3 manifests, which we don't yet support — so as expected, there will be some additional work needed before we can use these even if generating them works exactly as hoped.

**Comment by kmcelwee** [comment]
@rlskoeser So after about an hour of exploration:

Bodleian posts their XML on github! parsing the XML seems a lot easier than parsing the HTML presumably generated from this XML https://github.com/bodleian/genizah-mss/blob/f1190e06a074e8f3e6095b40cebed092afefe128/collections/MS_Heb_a_3.xml

The one problem is that they don't pair images with their metadata, so we'd have to. The mapping seems annoying but straightforward:

```
xml:id="MS_Heb_a_3-part4-item1" -> MS_HEB_a_3_4a.jpg 
```

But I'm seeing some scarily-named items I'm not exactly sure how to parse (e.g. `xml:id="MS_Heb_a_3-part10-item1-item1" `)

`biiif` seems like a really easy tool to use. I haven't tried it on bodleian data yet, but it's dead simple. For our purposes, I think just installing their CLI is all that's necessary: https://github.com/IIIF-Commons/biiif-cli

### Questions
* I'm not entirely sure about how IIIF structuring works. Would you mind double checking my proposed structure? I expect the IIIF collection would be "Bodleian volume 5" (i.e. the XML page), the manifest would be something like "MS. Heb. a. 3/45" and then canvases would be the items associated with each shelfmark (I think these are recto and verso). Is that about right?
* What kind of metadata is important to grab from the XML? Should I just start with the description and expand from there if necessary? Is this the metadata vocabulary I should be using? https://iiif.io/api/presentation/3.0/#metadata

**Comment by rlskoeser** [comment]
Wow, that's great we can get the data directly from the XML, agree that's much better.  Had not found the cli for biiif yet, glad there is one.

Yes, image names have to be determined based on shelfmark. Marina says she already has a set of the images, any easier to work from her copy than to grab them? We still have to generate the names from ids.

I want one collection per institution (i.e., bodleian). We need one manifest per shelfmark.

Basic metadata is fine — label, description, id; rights statement if they have something we can use.

**Comment by rlskoeser** [comment]
@kmcelwee how comfortable are you working with the xml?

**Comment by kmcelwee** [comment]
@rlskoeser I haven't worked with XML this complicated before, but for the basic things we need to extract, I'm comfortable, at least for now.

**Comment by rlskoeser** [comment]
@kmcelwee let's not pull from the xml yet. Please create a test collection with two manifests corresponding to two fragments in the PGP db. That way you can test biiif and biiif-cli work as described and we have sample manifests to test with.

**Comment by kmcelwee** [comment]
@rlskoeser Threw this together quickly: https://github.com/kmcelwee/geniza-671 You'd be better than me at double checking that this works as intended.

Also here's an example manifest being served by github pages:
https://kmcelwee.github.io/geniza-671/bodleian/ms-heb-a-3-1/index.json

**Comment by rlskoeser** [comment]
please test by loading in a demo version of mirador https://mirador-dev.netlify.app/__tests__/integration/mirador/

I'm getting errors with images not being found, were they not added properly?

**Comment by kmcelwee** [comment]
@rlskoeser Because of jekyll, directories starting with an underscore were ignored! I needed to add a file `.nojekyll` to the repo so that they were served. Also redirects were causing errors, since my site is not kmcelwee.github.io, but www.kmcelwee.com. It looks like it works now!

**Comment by rlskoeser** [comment]
making this an investigation chore and closing it; will open a new issue to track implementation as a feature/ user story
